src/types.ts

```diff
@@ -67,6 +67,7 @@ export interface Schema {
   uuid: string;
   encounterType: string;
   referencedForms: [];
+  version?: string;
```

Comment:
Neither here nor there, but for forms it might make sense to take the version from the Form object rather than the schema (it's required for the Form object; it's optional in the schema).

Reply:
When the user creates a new form (say by using the `input dummy schema` feature) and explicitly types in a version number in the schema editor, it wont get picked up by the form saving workflow if we don't provide `schema?.version || form?.version` as the possible values for the `Version` field. This isn't a problem when editing an existing form because the `version` gets picked from the form object.

Reply:
Yeah, that makes sense!